app/src/routes/api/v1/collection.router.js

```diff
@@ -14,6 +14,12 @@ class CollectionRouter {
 
     static async getAll(ctx) {
         logger.info('Obtaining collection by user');
+
+        if (!ctx.query.loggedUser) {
```

Comment:
BC break?

Reply:
Authentication is applied via a flag in the `register.json` file, so not a BC. This check is just so that an error does not get thrown when a couple of lines below we are accessing `ctx.query.loggedUser.id` without checking if it is defined :)